content/backmatter.tex

```diff
@@ -496,25 +519,61 @@ \subsection{\textit{Fortran} API}\label{subsec:deprecate-fortran} %% WARNING: Is
 which previously indicated subgroups of \acp{PE} with a list of
-parameters to describe the subgroup composition should be phased
+parameters to describe the subgroup composition (active set) should be phased
 out in favor of using collective operations with a team parameter.
 
-When moving from active set routines to teams based routines, the fixed-size
-versions of the routines, e.g. \FUNC{shmem\_broadcast32}, were not
-carried forward. Instead, all teams based collective routines use standard
+The library constants
+\begin{center}
+\begin{tabular}{ll}
+    \LibConstRef{SHMEM\_SYNC\_VALUE}            & \LibConstRef{SHMEM\_BCAST\_SYNC\_SIZE} \\
+    \LibConstRef{SHMEM\_SYNC\_SIZE}             & \LibConstRef{SHMEM\_COLLECT\_SYNC\_SIZE} \\
+    \LibConstRef{SHMEM\_BARRIER\_SYNC\_SIZE}    & \LibConstRef{SHMEM\_REDUCE\_SYNC\_SIZE} \\
+    \LibConstRef{SHMEM\_ALLTOALL\_SYNC\_SIZE}   & \LibConstRef{SHMEM\_REDUCE\_MIN\_WRKDATA\_SIZE} \\
+    \LibConstRef{SHMEM\_ALLTOALLS\_SYNC\_SIZE} \\
+\end{tabular}
+\end{center}
+were deprecated as these constants pertain only to active-set-based collectives.
+
+The \CorCpp active-set-based \FuncRef{shmem\_sync} routine was deprecated and
+replaced with the team-based \Cstd[11] \FuncRef{shmem\_sync} or \CorCpp
+\FuncRef{shmem\_team\_sync} routine.
+
+The fixed-sized versions of the active-set-based routines
+\begin{center}
+\begin{tabular}{ll}
+    \FuncRef{shmem\_alltoall32} & \FuncRef{shmem\_alltoall64} \\
+    \FuncRef{shmem\_alltoalls32} & \FuncRef{shmem\_alltoalls64} \\
+    \FuncRef{shmem\_broadcast32} & \FuncRef{shmem\_broadcast64} \\
+    \FuncRef{shmem\_collect32} & \FuncRef{shmem\_collect64} \\
+    \FuncRef{shmem\_fcollect32} & \FuncRef{shmem\_fcollect64} \\
+\end{tabular}
+\end{center}
+were deprecated. Instead, all team-based collective routines use standard
 \Cstd types with the option to use generic \Cstd[11] functions for more portable
 and maintainable implementations.
 
+The active-set-based reduction routines
+\begin{center}
+\begin{tabular}{ll}
+    \FuncRef{shmem\_\FuncParam{TYPENAME}\_and\_to\_all} & \FuncRef{shmem\_\FuncParam{TYPENAME}\_max\_to\_all} \\
+    \FuncRef{shmem\_\FuncParam{TYPENAME}\_or\_to\_all}  & \FuncRef{shmem\_\FuncParam{TYPENAME}\_min\_to\_all} \\
+    \FuncRef{shmem\_\FuncParam{TYPENAME}\_xor\_to\_all} & \FuncRef{shmem\_\FuncParam{TYPENAME}\_sum\_to\_all} \\
+                                                        & \FuncRef{shmem\_\FuncParam{TYPENAME}\_prod\_to\_all} \\
+\end{tabular}
+\end{center}
+were deprecated and replaced with team-based reduction routines.
+
+
 \subsection{\CorCpp: \FUNC{shmem\_barrier}}
 Each \openshmem team might
 be associated with some number of communication contexts. The \FUNC{shmem\_barrier}
-functions imply that the default context is quiesced after synchronizing
-some set of \acp{PE}. Since teams may have some number of contexts associated
+function implies that the default context is quiesced after synchronizing
```

Comment:
Minor thing:  my understanding of the original text is that both `shmem_barrier` and `shmem_barrier_all` exhibit this behavior, but I'm not sure whether this change fully captures that...

Reply:
The description for `shmem_barrier_all` does have a Note that says it is equivalent to quiet + sync.

Unless I'm mistaken, for this paragraph, I decided to refocus the sentence on only `shmem_barrier` because the rest of the paragraph explains the ambiguity of implicitly deciding which context (in a given team) to quiesce. For `shmem_barrier_all`, there is no ambiguity because `SHMEM_TEAM_WORLD` will be quiesced through `SHMEM_CTX_DEFAULT`.

I don't mind making a change / reversing this change if you think the sentence could be worded better.

Reply:
Good point - I have no problem either way.  I was initially thinking this could technically be a semantic change, but it seems like you're probably covered.